### PR TITLE
Update updating.md

### PR DESCRIPTION
**Pull request by JamesH65**
dist-upgrade explanation

### CODE REVIEW

**Comment by JamesH65** [comment]
I wonder if we should also change apt-get to apt at this point?

**Comment by XECDesign** [comment]
> I wonder if we should also change apt-get to apt at this point?

Yup

**Comment by JamesH65** [comment]
Oh, goodo, not too many then!! I'll grab a full copy of the tree and do a global replace in a separate PR.

**Comment by JamesH65** [comment]
See #1310

File: raspbian/updating.md [hunk 1]
```diff
@@ -36,7 +38,7 @@ The kernel and firmware are installed as a Debian package, and so will also get
 
 ### Running out of space
 
-When running `sudo apt-get dist-upgrade`, it will show how much data will be downloaded and how much space it will take up on the SD card. It's worth checking with `df -h` that you have enough free disk space, as unfortunately `apt` will not do this for you. Also be aware that downloaded package files (`.deb` files) are kept in `/var/cache/apt/archives`. You can remove these in order to free up space with `sudo apt-get clean`.
+When running `sudo apt dist-upgrade`, it will show how much data will be downloaded and how much space it will take up on the SD card. It's worth checking with `df -h` that you have enough free disk space, as unfortunately `apt` will not do this for you. Also be aware that downloaded package files (`.deb` files) are kept in `/var/cache/apt/archives`. You can remove these in order to free up space with `sudo apt clean`.
```
Comment (XECDesign) [inline raspbian/updating.md]: One difference in behaviour between `apt` and `apt-get` is that `apt` will remove the downloaded archives afterwards. So the last two sentences are not necessary.

**Comment by lurch** [comment]
Should this be modified to use `full-upgrade` too, @XECDesign ?

**Comment by XECDesign** [comment]
I'm guessing that since this PR will conflict with #1310, it should probably all just be done there.

File: raspbian/updating.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ sudo apt update
 Next, **upgrade** all your installed packages to their latest versions with the following command:
 
 ```bash
-sudo apt dist-upgrade
+sudo apt full-upgrade
 ```
 
 Note that `dist-upgrade` is used in preference to a simple `upgrade`, as it also picks up any dependency changes that may have been made.
```
Comment (lurch) [inline raspbian/updating.md]: I guess this line should have been changed to `full-upgrade` too? :eyes:

Comment (JamesH65) [reply raspbian/updating.md]: No worries, I only spotted it on a proof read before merging!

**Pull request merged**